velox/dwio/parquet/reader/duckdb/InputStreamFileSystem.h

```diff
@@ -39,152 +45,101 @@ class InputStreamFileSystem : public ::duckdb::FileSystem {
+    VELOX_NYI();
   }
 
   int64_t GetFileSize(::duckdb::FileHandle& handle) override {
-    auto& streamHandle = dynamic_cast<InputStreamFileHandle&>(handle);
-    dwio::common::InputStream* stream = nullptr;
-    {
-      auto lockedStreams = streams_.rlock();
-      auto it = lockedStreams->find(streamHandle.streamId());
-      VELOX_CHECK(
-          it != lockedStreams->end(),
-          "Unknown stream with ID {}",
-          streamHandle.streamId());
-      stream = it->second.second.get();
-    }
-    return stream->getLength();
+    return stream_->getLength();
   }
 
   time_t GetLastModifiedTime(::duckdb::FileHandle& /*handle*/) override {
-    VELOX_FAIL("Unexpected call to InputStreamFileSystem::GetLastModifiedTime");
+    VELOX_NYI();
   }
 
   void Truncate(::duckdb::FileHandle& /*handle*/, int64_t /*new_size*/)
       override {
-    VELOX_FAIL("Unexpected call to InputStreamFileSystem::Truncate");
+    VELOX_NYI();
   }
 
   bool DirectoryExists(const std::string& /*directory*/) override {
-    VELOX_FAIL("Unexpected call to InputStreamFileSystem::DirectoryExists");
+    VELOX_NYI();
   }
 
   void CreateDirectory(const std::string& /*directory*/) override {
-    VELOX_FAIL("Unexpected call to InputStreamFileSystem::CreateDirectory");
+    VELOX_NYI();
   }
 
   void RemoveDirectory(const std::string& /*directory*/) override {
-    VELOX_FAIL("Unexpected call to InputStreamFileSystem::RemoveDirectory");
+    VELOX_NYI();
   }
 
   bool ListFiles(
       const std::string& /*directory*/,
       const std::function<void(std::string, bool)>& /*callback*/) override {
-    VELOX_FAIL("Unexpected call to InputStreamFileSystem::ListFiles");
+    VELOX_NYI();
   }
 
   void MoveFile(const std::string& /*source*/, const std::string& /*target*/)
       override {
-    VELOX_FAIL("Unexpected call to InputStreamFileSystem::MoveFile");
+    VELOX_NYI();
   }
 
   bool FileExists(const std::string& /*filename*/) override {
-    VELOX_FAIL("Unexpected call to InputStreamFileSystem::FileExists");
+    VELOX_NYI();
   }
 
   void RemoveFile(const std::string& /*filename*/) override {
-    VELOX_FAIL("Unexpected call to InputStreamFileSystem::RemoveFile");
+    VELOX_NYI();
   }
 
   void FileSync(::duckdb::FileHandle& /*handle*/) override {
-    VELOX_FAIL("Unexpected call to InputStreamFileSystem::FileSync");
+    VELOX_NYI();
   }
 
   std::vector<std::string> Glob(const std::string& /*path*/) override {
-    VELOX_FAIL("Unexpected call to InputStreamFileSystem::Glob");
+    VELOX_NYI();
   }
 
   virtual std::string GetName() const override {
-    VELOX_FAIL("Unexpected call to InputStreamFileSystem::GetName");
+    VELOX_NYI();
   }
 
  private:
-  std::atomic<int> nextStreamId_ = 1;
-  // Maps stream ID to file handle counter and input stream.
-  folly::Synchronized<folly::F14FastMap<
-      int,
-      std::pair<int, std::unique_ptr<dwio::common::InputStream>>>>
-      streams_;
+  std::unique_ptr<dwio::common::InputStream> stream_;
```

Comment:
Earlier, this object used to manage multiples streams/file-io. Now it doesn't ? Is that understanding correct ?